curp/src/client.rs

```diff
@@ -251,6 +250,8 @@ where
         mut new_leader: Option<ServerId>,
     ) -> Result<(), ProposeError> {
         loop {
+            tokio::time::sleep(Self::RETRY_INTERVAL).await;
```

Comment:
I think we'd better use `ClientTimeout::retry_timeout` to replace `Self::RETRY_INTERVAL`.

Reply:
Pending on #177